src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -147,7 +147,8 @@ protected virtual void Initialize()
             }
             else
             {
-                TraceWriter = NullTraceWriter.Instance;
+                // if no TraceWriter has been configured, default it to Console
+                TraceWriter = new ConsoleTraceWriter(hostTraceLevel);
```

Comment:
For the host TraceWriter in non File logging cases, rather than ignore host output, write to Console. This facilitates command line ScriptHost debugging. In the context of this review, it means that I can see the errors written when the schedule expression is invalid.